docs/kb/release-notes.md

```diff
@@ -0,0 +1,29 @@
+# Release Notes
+
+Here you'll find the latest updates on what's changed in the Pardot API and API documentation.
```

Comment:
This title contains several contractions and slang. Best to avoid such in tech writing. 
Perhaps "This page contains the release notes for the Pardot API and related documentation."